finalPrep/modZeroHero.js

```diff
@@ -1,50 +1,74 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
-
+var heroName = "Red Lantern";
+var specialAbility = "Faster than the speed of light";
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
-
+var greeting = "Hi! You're safe now!";
```

Comment:
These directions wanted you to show how to use both concatenation and interpolation. I saw other examples of this throughout the rest of your work though!